syft/frameworks/torch/tensors/interpreters/multi_pointer.py

```diff
@@ -17,7 +18,7 @@ def __init__(
         shape=None,
         point_to_attr=None,
         tags=None,
-        description=None,
+        description: str = None,
         children=None,
```

Comment:
maybe you want to add annotation for all arguments (whe it's not too painful) or for the whole file, this way people continuing the type annotation know they don't have to check everything
https://docs.python.org/3/library/typing.html can be a good resource